ros/src/computing/planning/motion/packages/op_simulation_package/launch/op_car_simulator_2.launch

```diff
@@ -0,0 +1,116 @@
+		<param name="baseColorR" 					value="$(arg baseColorR)" />
+		<param name="baseColorG" 					value="$(arg baseColorG)" />
+		<param name="baseColorB" 					value="$(arg baseColorB)" />
+		<param name="logFolder" 					value="$(arg logFolder)" />			
+		
+		<param name="maxVelocity" 					value="$(arg maxVelocity)" />
+	    <param name="minVelocity" 					value="$(arg minVelocity)" />
+	    	    		
+		<param name="maxLocalPlanDistance" 			value="$(arg maxLocalPlanDistance)" />
+		<param name="samplingTipMargin" 			value="$(arg samplingTipMargin)" />
+		<param name="samplingOutMargin" 			value="$(arg samplingOutMargin)" />
+		<param name="samplingSpeedFactor" 			value="$(arg samplingSpeedFactor)" />
+		<param name="pathDensity" 					value="$(arg pathDensity)" />
+		<param name="rollOutDensity" 				value="$(arg rollOutDensity)" />
+		<param name="rollOutsNumber" 				value="$(arg rollOutsNumber)" />
+		<param name="horizonDistance" 				value="$(arg horizonDistance)" />
+		
+		<param name="minFollowingDistance" 			value="$(arg minFollowingDistance)" />		
+		<param name="minDistanceToAvoid" 			value="$(arg minDistanceToAvoid)" />
+		<param name="maxDistanceToAvoid" 			value="$(arg maxDistanceToAvoid)" />
+		<param name="speedProfileFactor"			value="$(arg speedProfileFactor)" />
+		
+		<param name="horizontalSafetyDistance"		value="$(arg horizontalSafetyDistance)" />
+		<param name="verticalSafetyDistance"		value="$(arg verticalSafetyDistance)" />
+		
+		<param name="enableSwerving" 				value="$(arg enableSwerving)" />
+		<param name="enableFollowing" 				value="$(arg enableFollowing)" />
+		<param name="enableHeadingSmoothing" 		value="$(arg enableHeadingSmoothing)" />
+		<param name="enableTrafficLightBehavior" 	value="$(arg enableTrafficLightBehavior)" />
+		<param name="enableStopSignBehavior" 		value="$(arg enableStopSignBehavior)" />		
+		<param name="enableLaneChange" 				value="$(arg enableLaneChange)" />		
+		
+		<param name="width" 						value="$(arg width)" />
+		<param name="length" 						value="$(arg length)" />
+		<param name="wheelBaseLength" 				value="$(arg wheelBaseLength)" />
+		<param name="turningRadius" 				value="$(arg turningRadius)" />
+		<param name="maxSteerAngle" 				value="$(arg maxSteerAngle)" />
+		
+		<param name="steeringDelay" 				value="$(arg steeringDelay)" />
+		<param name="minPursuiteDistance" 			value="$(arg minPursuiteDistance)" />
+		
+		<param name="maxAcceleration" 				value="$(arg maxAcceleration)" />
+		<param name="maxDeceleration" 				value="$(arg maxDeceleration)" />
+		
+		<param name="enableCurbObstacles" 			value="$(arg enableCurbObstacles)" />
+		<param name="enableObjectTracking" 			value="$(arg enableObjectTracking)" />
+		<param name="enableOutsideControl" 			value="$(arg enableOutsideControl)" />
+				
+		<param name="mapSource" 					value="$(arg mapSource)" />
+		<param name="mapFileName" 					value="$(arg mapFileName)" />
+		
+	</node>
+
+</launch>
```

Comment:
some documentation what all of these params do would be great

Reply:
I totally agree with you :)